docs/docs/configuration/index.md

```diff
@@ -3,9 +3,11 @@ id: index
 title: Frigate Configuration
 ---
 
-For Home Assistant Addon installations, the config file needs to be in the root of your Home Assistant config directory (same location as `configuration.yaml`). It can be named `frigate.yaml` or `frigate.yml`, but if both files exist `frigate.yaml` will be preferred and `frigate.yml` will be ignored.
+For Home Assistant Add-on installations, the config file should be at `/addon_configs/<addon_directory>/config.yaml`, where `<addon_directory>` is specific to the variant of the Frigate Add-on you are running. See the list of directories [here](#accessing-add-on-config-dir).
 
-For all other installation types, the config file should be mapped to `/config/config.yml` inside the container.
+For all other installation types, the config file should be mapped to `/config/config.yaml` inside the container.
+
+It can be named `config.yaml` or `config.yml`, but if both files exist `config.yaml` will be preferred and `config.yml` will be ignored.
```

Comment:
> t can be named `config.yaml` or `config.yml`, but if both files exist `config.yaml` will be preferred and `config.yml` will be ignored.

Is this still true? @NickM-27

Reply:
From what I remember from the scripts, yeah. I'm not home now, but for example you can check the go2rtc create_config.py.

Reply:
Here:

https://github.com/felipecrs/frigate/blob/a0c8cc9562ebf5ebeeee389a79824809adf9ff4b/docker/main/rootfs/usr/local/go2rtc/create_config.py#L35-L37

yaml is preferred.

Reply:
ah, this is actually no longer true, .yml is preferred. We should update the go2rtc create_config.py to use this function as well

https://github.com/blakeblackshear/frigate/blob/e36fe797828e695520d58faccf4dd7397deabaf3/frigate/util/config.py#L20

Reply:
Should I also drop support for loading `config.json`? Or users can still set CONFIG_FILE to a json file?

Reply:
My vote for that would be a yes, since writing back to the config from the UI is not supported with a json config. Let's see what @NickM-27 says.

Reply:
I believe changing the env variable to json will work, best to leave it the same for now